mscore/file.cpp

```diff
@@ -493,8 +493,7 @@ void MuseScore::updateNewWizard()
 
 void MuseScore::newFile()
       {
-      if (newWizard == 0)
-            newWizard = new NewWizard(this);
+      newWizard->updateNewWizard();
```

Comment:
not "else"?

Reply:
No else needed, it is all done inside `updateNewWizard()` already

Reply:
the `newWizard->` was wrong though